h/models/annotation.py

```diff
@@ -38,6 +38,11 @@ class Annotation(Base):
         types.URLSafeUUID, server_default=sa.func.uuid_generate_v1mc(), primary_key=True
     )
 
+    #: Annotation PK: this was added after annotation.id and it's meant to replace it short term
+    #: For now we'll have a sequence and a nullable serial column.
+    pk_sequence = sa.Sequence("annotation_id_seq", cycle=False)
+    pk = sa.Column(sa.Integer, nullable=True, unique=True)
```

Comment:
This is not the table's PK but it's call PK here.

`annotation.id` is taken by the UUID column. I reckon ideally we call that `annotation.uuid` or `annotation.pubid` but I don't think we should rename that until the new column is the pk.

We could in the interim name this `annotation.private_id` or similar but that would be inconsistent with all the other `table.id` columns.

`annotation.pk` I reckon is the best for now.